engine/classes/ElggInstaller.php

```diff
@@ -356,6 +356,12 @@ protected function database($submissionVars) {
 				'value' => 'elgg_',
 				'required' => TRUE,
 				),
+			'timezone' => array(
+				'type' => 'dropdown',
+				'value' => 'UTC',
+				'options' => \DateTimeZone::listIdentifiers(),
+				'required' => TRUE
+			)
```

Comment:
I think this should be user specific instead of site specific setting. Servers should always use UTC to avoid trouble related to timezone differences, daylight saving time, etc.

Setting the target timezone directly is of of course sufficient for sites used only within single timezone, but still I'd prefer adding a proper user specific timezone support instead.